index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+       <a class="name">John Doe</a>
+        <div class="hamburger"> 
+            <img src="icons/Union.png" alt="Hamburger menu icon">
+        </div>
```

Comment:
Even if now we cannot see it, the hamburger button will hold the navigation.

- [ ] Therefore, in order to use the semantic HTML tags, kindly, replace the `div` with a `nav`;